### PR TITLE
Rename all occurences of Master/Detail to Overview/Detail.

### PR DESCRIPTION
**Pull request by zach-klippenstein**
Fixes the kotlin half of #1188.

## Checklist

- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: kotlin/samples/containers/android/src/main/res/layout-ldrtl/overview_detail_split.xml [hunk 1]
```diff
@@ -21,10 +21,10 @@
     >
 
   <com.squareup.workflow.ui.WorkflowViewStub
-      android:id="@+id/master_stub"
+      android:id="@+id/detail_stub"
```
Comment (rjrjr) [inline kotlin/samples/containers/android/src/main/res/layout-ldrtl/overview_detail_split.xml:24]: These look backward.

File: kotlin/samples/containers/android/src/main/res/layout/overview_detail_split.xml [hunk 2]
```diff
@@ -21,10 +21,10 @@
     >
 
   <com.squareup.workflow.ui.WorkflowViewStub
-      android:id="@+id/detail_stub"
+      android:id="@+id/overview_stub"
```
Comment (rjrjr) [inline kotlin/samples/containers/android/src/main/res/layout/overview_detail_split.xml:24]: Backward again.

File: kotlin/samples/containers/android/src/main/res/layout-ldrtl/overview_detail_split.xml [hunk 3]
```diff
@@ -34,10 +34,10 @@
       />
 
   <com.squareup.workflow.ui.WorkflowViewStub
-      android:id="@+id/detail_stub"
+      android:id="@+id/overview_stub"
```
Comment (rjrjr) [inline kotlin/samples/containers/android/src/main/res/layout-ldrtl/overview_detail_split.xml:37]:
```suggestion
      android:id="@+id/detail_stub"
```

File: kotlin/samples/containers/android/src/main/res/layout/overview_detail_split.xml [hunk 4]
```diff
@@ -34,10 +34,10 @@
       />
 
   <com.squareup.workflow.ui.WorkflowViewStub
-      android:id="@+id/master_stub"
+      android:id="@+id/detail_stub"
```
Comment (rjrjr) [inline kotlin/samples/containers/android/src/main/res/layout/overview_detail_split.xml:37]:
```suggestion
      android:id="@+id/overview_stub"
```

File: kotlin/samples/containers/android/src/main/res/layout/overview_detail_split.xml [hunk 5]
```diff
@@ -21,10 +21,10 @@
     >
 
   <com.squareup.workflow.ui.WorkflowViewStub
-      android:id="@+id/detail_stub"
+      android:id="@+id/overview_stub"
       android:layout_width="0dp"
       android:layout_height="match_parent"
-      android:layout_weight="70"
+      android:layout_weight="30"
```
Comment (zach-klippenstein) [inline kotlin/samples/containers/android/src/main/res/layout/overview_detail_split.xml:27]: Not sure what happened here either

**Comment by zach-klippenstein** [comment]
@rjrjr Ok, I think what happened was that Git just got confused, and instead of detecting the changes to the split layouts within their LTR/RTL directories, detected moves. It thinks the LTR file moved to RTL and vice-versa, which is why the contents look like they flipped. But I just verified that the layouts actually have correct layout direction.

**Pull request merged**